examples/python/visualization/tensorboard_tensorflow.py

```diff
@@ -33,8 +33,9 @@
 import tensorflow as tf
 
 BASE_LOGDIR = "demo_logs/tf/"
-MODEL_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..',
```

Comment:
Just a comment: I thought the original one is more concise, what is the advantage of using `os.path.dername`? If we use `pathlib`, it will be called `parent`, which is also good. It's fine to keep it as it is. We'll do another round after the dataset class is in place.

Reply:
I'm not really sure what the advantage is of using `os.path.dirname`. I was trying to keep it consistent with #4562.